.github/workflows/set-backlog-fields.yml

```
@@ -0,0 +1,103 @@
+name: Add Issue to Project
+
+on:
+  issues:
+    types: [ opened ]
+
+jobs:
+  steve:
+    if: ${{ !github.event.issue.pull_request }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@3bc28d706f1f8026f2d39c8fc790d4336bb1cf7e
+        with:
+          app_id: ${{ secrets.LFPROJECTBOARDAUTOMATION_APP_ID }}
+          private_key: ${{ secrets.LFPROJECTBOARDAUTOMATION_PRIVATE_KEY }}
+          
+      - name: determine if issue is already on a project
+        env:
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+          ISSUE_NUMBER: ${{ github.event.issue.number }}
+          REPO: ${{ toJson(github.event.repository.name) }}
+          ORGANIZATION: sillsdev
+        run: |
+          echo issue $ISSUE_NUMBER added to $REPO
+          
+          gh api graphql -f query='
+            query($issue_number:Int!, $org:ID!) {
+              repository(name:"web-languageforge", owner:$org) {
```

```suggestion
              repository(name:"${REPO}", owner:$org) {
```
I'm not actually sure this will work given the single and double quote acrobatics here, but I thought it was worth a suggestion.

---

That's a good idea, since I might want to copy this workflow into the LfMerge repo.

---

Yeah, it should definitely be noted that the task in its current state applies to "web-languageforge" project "1" and nothing else right now. I plan to change the repo for the final state, but the project number will be slightly more tricky if we start incrementing that.